schemavalidator/schemavalidator.py

```diff
@@ -130,6 +130,9 @@ def validate(self, document, schema_id):
         except ValidationError as e:
             raise SchemaValidationError(e.message) from e
 
+    def validate_string(self, document, schema_id):
```

Comment:
I think `validate_json` is a better description of the purpose of the function. May also be a good idea to rename the `document` argument, something like `json_string`.